### PR TITLE
refactor: pipeline graph to pb provider

### PR DESCRIPTION
**Pull request by chengjoey**
#### What this PR does / why we need it:
refactor pipeline graph service to provider

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |    refactor pipeline graph service to provider          |
| 🇨🇳 中文    |     将pipeline的图形展示接口重构成provider         |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: internal/tools/pipeline/providers/graph/graph.service.go [hunk 1]
```diff
@@ -12,40 +12,62 @@
...
+	"github.com/erda-project/erda-proto-go/core/pipeline/graph/pb"
 	"github.com/erda-project/erda/apistructs"
+	"github.com/erda-project/erda/bundle"
 	"github.com/erda-project/erda/internal/tools/pipeline/services/apierrors"
 	"github.com/erda-project/erda/pkg/i18n"
 	"github.com/erda-project/erda/pkg/parser/pipelineyml"
 	"github.com/erda-project/erda/pkg/strutil"
 )
 
-func (s *PipelineSvc) PipelineYmlGraph(req *apistructs.PipelineYmlParseGraphRequest) (*apistructs.PipelineYml, error) {
```
Comment (sfwn) [inline internal/tools/pipeline/providers/graph/graph.service.go:28]: `apistructs.PipelineYmlParseGraphRequest` should be deleted

File: pkg/parser/pipelineyml/define.go [hunk 2]
```diff
@@ -36,9 +38,9 @@ type Spec struct {
 
 	Name string `yaml:"name"`
 
-	On       *TriggerConfig                `yaml:"on,omitempty"`
-	Triggers []*apistructs.PipelineTrigger `yaml:"triggers,omitempty"` // todo Solve the problem that quotation marks will be automatically added after yaml is saved
```
Comment (sfwn) [inline pkg/parser/pipelineyml/define.go:40]: `apistructs.PipelineTrigger` should be deleted

File: pkg/parser/pipelineyml/graph_pipelineyml_test.go [hunk 3]
```diff
@@ -46,12 +46,12 @@ func TestConvertToGraphPipelineYml(t *testing.T) {
 
 func Test_cronCompensatorReset(t *testing.T) {
 	type args struct {
-		cronCompensator *apistructs.CronCompensator
```
Comment (sfwn) [inline pkg/parser/pipelineyml/graph_pipelineyml_test.go:49]: and this

**Review by sfwn — CHANGES_REQUESTED** [review]

File: internal/tools/pipeline/providers/graph/graph.service.go (hunk 1 above)
Comment (chengjoey) [reply internal/tools/pipeline/providers/graph/graph.service.go:28]: done

File: pkg/parser/pipelineyml/define.go (hunk 2 above)
Comment (chengjoey) [reply pkg/parser/pipelineyml/define.go:40]: done

File: pkg/parser/pipelineyml/graph_pipelineyml_test.go (hunk 3 above)
Comment (chengjoey) [reply pkg/parser/pipelineyml/graph_pipelineyml_test.go:49]: done

**Review by sfwn — APPROVED** [review]

**Review by sfwn — APPROVED** [review]

**Pull request merged**